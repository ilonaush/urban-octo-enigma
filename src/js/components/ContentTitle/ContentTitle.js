import React from 'react';
import "./ContentTitle.styl";

const ContentTitle = ({children, style = {}, ...props}) => (
    <h3 className='content-title'>{children}</h3>
);

export default ContentTitle;
