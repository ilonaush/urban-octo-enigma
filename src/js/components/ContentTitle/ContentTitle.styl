@import "~js/vars"

.content-title
  color $orange
  font-family $montserrat-thin
  padding 20px
  font-weight 100


