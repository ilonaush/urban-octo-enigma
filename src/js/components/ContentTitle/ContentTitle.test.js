import React from 'react';
import ReactDOM from 'react-dom';
import ContentTitle from './ContentTitle';

it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<ContentTitle />, div);
});