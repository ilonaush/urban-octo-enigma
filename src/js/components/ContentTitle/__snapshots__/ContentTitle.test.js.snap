// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentTitle renders correctly 1`] = `
<h3
  className="content-title"
/>
`;

exports[`ContentTitle renders correctly without children 1`] = `
<h3
  className="content-title"
/>
`;
